k8s/charts/openebs/values.yaml

```diff
@@ -52,9 +52,9 @@ provisioner:
 localprovisioner:
   enabled: true
   image: "quay.io/openebs/provisioner-localpv"
-  imageTag: "1.4.0"
-  helperImage: "quay.io/openebs/openebs-tools"
-  helperImageTag: "3.8"
+  imageTag: "1.5.0-RC2"
+  helperImage: "quay.io/openebs/linux-utils"
+  helperImageTag: "1.4.0"
```

Comment:
use 1.5.0

Reply:
add this to the list of configuration options in the readme.md

Reply:
can you create a different section for helper image, similar to exporter. And use that for local pv and ndm. 
also modify the maya apiserver env variables to include OPENEBS_IO_HELPER_IMAGE

Reply:
Implemented the changes requested.